docs/rules/object-curly-newline.md

```diff
@@ -19,19 +19,23 @@ Or an object option:
 * `"minProperties"` requires line breaks if the number of properties is at least the given integer. By default, an error will also be reported if an object contains linebreaks and has fewer properties than the given integer. However, the second behavior is disabled if the `consistent` option is set to `true`
 * `"consistent": true` requires that either both curly braces, or neither, directly enclose newlines. Note that enabling this option will also change the behavior of the `minProperties` option. (See `minProperties` above for more information)
 
-You can specify different options for object literals and destructuring assignments:
+You can specify different options for object literals and destructuring assignments, named imports and exports:
```

Comment:
Sorry, I wasn't clear in my last comment. The removal of the original "and" was correct, but I wanted a new "and" before the named import/export.

This is what I was going for:

> You can specify different options for object literals, destructuring assignments, and named imports and exports:

Thanks for your patience!